flask/temp_comparator.txt

```diff
@@ -0,0 +1,119 @@
+Baits EM MGifts
```

Comment:
It looks like this file is just temp? It probably should be either part of gitignore or just use a tempfile https://docs.python.org/3.6/library/tempfile.html

Reply:
It's not exactly a regular temp file that can be ignored—the way that the diff works is that it takes in two files to compare. One file is the regular Meal/LocationMain file we already have, but the other we need to create and insert in the updated Meals/Location nightly to compare with the preexisting Meal/LocationMain. Obviously there are methods that don't need this but it makes the diff process easier to work with and log changes.